plugin/teksi_wastewater/interlis/gui/interlis_importer_exporter_gui.py

```diff
@@ -113,6 +113,8 @@ def action_export(self):
                 export_models=export_dialog.selected_models(),
                 logs_next_to_file=export_dialog.logs_next_to_file,
                 limit_to_selection=export_dialog.limit_to_selection,
+                # export orientation added
+                export_orientation=export_dialog.selected_orientation,
```

Comment:
```suggestion
                export_orientation=export_dialog.selected_orientation(),
```

Or you could add the `@property` decoration like for logs_next_to_file, limit_to_selection, etc...

Reply:
@domi4484 I will go for  `@property` - thanks for all the hints.